### ISSUE TITLE
Slimes take poison damage from CO2

### ISSUE THREAD

**Issue by K-Dynamic** [space-wizards/space-station-14#28970]
## Description
Slimes (despite not processing oxygen) take poison damage from CO2. 

It should be noted vox are immune to CO2. However, they are poisoned by oxygen instead, and are not immune to sleep from NO2.

**Reproduction**
1. Be slime
2. Replace all the air with CO2
3. Get poisoned

A 'fix' would be just duplicating the yaml line entry that makes vox immune to CO2 for slimes, however it might be worth discussing if slimes *should* be immune to CO2 too based on the vox conjecture - when really they're biologically different.

**Comment by dragonryan06** [comment]
If humans breathe O2, exhale CO2, are made sleepy by NO2, and poisoned by CO2

Shouldn't slimes breathe N2, exhale N2O, be made sleepy by CO2, and poisoned by N2O?

just my thoughts on it

**Comment by superjj18** [comment]
> If humans breathe O2, exhale CO2, are made sleepy by N2O, and poisoned by CO2

Humans are not poisoned by co2, you're confusing carbon dioxide with carbon monoxide(co).  Humans simply can't breath co2 so they asphyxiate(airloss damage) in high co2 environments such as fires.

(Oh jeez didn't notice how old this issue is, but slimes still take damage by co2)

**Comment by K-Dynamic** [comment]
> Humans are not poisoned by co2, you're confusing carbon dioxide with carbon monoxide(co). Humans simply can't breath co2 so they asphyxiate(airloss damage) in high co2 environments.

Yeah I noticed that too, unfortunately it seems SS14 just lumps the two together.
